### PR TITLE
style(frontend): update modals style

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Modal style updated as per requirements.

### Before

Screenshot 2024-09-09 at 14 10 40
Screenshot 2024-09-09 at 14 10 47
Screenshot 2024-09-09 at 14 10 53

### After

Screenshot 2024-09-09 at 14 09 37
Screenshot 2024-09-09 at 14 09 44
Screenshot 2024-09-09 at 14 09 48

### Mobile

Screenshot 2024-09-09 at 14 20 02
Screenshot 2024-09-09 at 14 20 28

### CODE REVIEW

**Comment by peterpeterparker** [comment]
Still looks good on small/mobile devices? Can you provide a screnshot?

**Comment by AntonioVentilii-DFINITY** [comment]
> Still looks good on small/mobile devices? Can you provide a screnshot?

Included mobile screenshots in the description

**Comment by AntonioVentilii-DFINITY** [comment]
@artkorotkikh-dfinity please have a look

File: src/frontend/src/lib/styles/global/gix.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	--value-color: var(--color-dark);
 
 	--border-radius-2x: var(--border-radius-lg);
-	--border-radius-0_5x: calc(0.5 * var(--border-radius-sm));
+	--border-radius-0_5x: var(--border-radius-xs);
```
Comment (peterpeterparker) [inline src/frontend/src/lib/styles/global/gix.scss:18]: This change impacts the overall app - i.e. other components that the modal, such as tags, progress steps etc. This is expected?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/styles/global/gix.scss:18]: that is just a small simplification that i put there:

`--border-radius-xs` by definition is `calc(0.5 * var(--border-radius-sm))`

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**